packages_rs/nextclade/src/align/align.rs

```diff
@@ -66,16 +66,28 @@ pub fn align_nuc(
 
   let mut attempt = 0;
 
+  let mut pre_retry_alignment: Option<AlignmentOutput<Nuc>> = None;
```

Comment:
The uninitialized variables is a code smell and a sign of either bugs or poorly structured code. 

On the other hand, I don't see immediately if this can be avoided with how the initial loop is written.

Does it mean we persist previous alignment result (including sequences), increasing memory consumption?

Reply:
We don't really need to keep the alignment around once the stripes return successfully, so we could empty it again.

Does the alignment take much memory? It's not the stripes actual memory-costly scores/paths that are kept around here, only the result.